### PR TITLE
WCSS: Add relocated docs

### PR DESCRIPTION
**Pull request by blackfalcon**
# Alaska Airlines Pull Request

**Fixes:** https://github.com/AlaskaAirlines/WC-Generator/issues/90

## Summary:

This PR will update the WCSS repo with relocated docs from the Auro repo. 

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [x] Other (docs)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: documents/shadyCSS.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  padding: var(--size-scale-xl) var(--size-scale-xl) var(--size-scale-sml) 0;
+  position: relative;
+  -webkit-transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  -o-transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  width: 100%;
+  font-size: var(--size-breakpoint-all-med);
+  caret-color: #0074cb;
+  caret-color: var(--color-brand-blue-atlas-base);
+  background-color: transparent;
```
Comment (jason-capsule42) [inline documents/shadyCSS.md:28]: Let's make this compliant with our styleguide sorting rules

File: documents/shadyCSS.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  padding: 0; /* affected */
+  position: relative;
+  -webkit-transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  -o-transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  width: 100%;
+  font-size: ; /* affected */
+  caret-color: #0074cb;
+  caret-color: ;
+  background-color: transparent;
```
Comment (jason-capsule42) [inline documents/shadyCSS.md:51]: Let's make this compliant with our style guide sorting rules

File: documents/shadyCSS.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  border-radius: 0;
+  border-bottom-width: var(--border-width-thin);
+  padding: var(--size-scale-xl) var(--size-scale-xl) var(--size-scale-sml);
+  padding-left: 0;
+  position: relative;
+  transition: all 0.3s cubic-bezier(0.215, 0.61, 0.355, 1);
+  width: 100%;
+  font-size: var(--size-breakpoint-all-med);
+  caret-color: var(--color-brand-blue-atlas-base);
+  background-color: transparent;
```
Comment (jason-capsule42) [inline documents/shadyCSS.md:72]: Let's make this compliant with our style guide sorting rules

File: documents/customProperties.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+CSS custom properties, aka CSS variables, are only supported by current browsers.
+
+| Browser | Supported release | Release date | Engine |
+|---|---|---|---|
+| Chrome	| 49 | March 2, 2016 | Blink |
+| Safari	| 9.1 | March 20, 2016 | Webkit |
+| Firefox | 31 | July 21, 2014 | Gecko |
+| Edge		| 16 | October 16, 2017 | EdgeHTML |
+| Opera 	| 36 | March 6, 2016 | Blink |
+| Internet Explorer | no support | no support | Trident |
```
Comment (jason-capsule42) [inline documents/customProperties.md:12]: Can we line up these columns?

Comment (blackfalcon) [reply documents/customProperties.md:12]: I usually find that to be a fool's game. Depending on the font you are using, trying to get things to line up will never be the same. 

For example, the code snapshot above looks totally different than it does in my IDE.

**Comment by blackfalcon** [comment]
@jason-capsule42 branch is rebased, please let me know if there are any issues.

**Comment by blackfalcon** [comment]
@jason-capsule42 if we could get an 👍  on this and then I can close https://github.com/AlaskaAirlines/WC-Generator/issues/90

File: documents/customProperties.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+```css
+:host {
+  --base-color: #ff000;
+}
+
+.selector {
+  color: var(--base-color);
+}
+```
```
Comment (jason-capsule42) [inline documents/customProperties.md:40]: The way I read this I expected `:root {}` to also be inside the code sample.

Comment (jason-capsule42) [reply documents/customProperties.md:40]:
```css
:root {
  --base-color: #00000;
}

:host {
  --base-color: #ff000;
}
.selector {
  color: var(--base-color);
}
```

Comment (blackfalcon) [reply documents/customProperties.md:40]: No. The `root` selector is higher up the CSSOM tree. This is illustrating how a `host` selector will overwrite the global `root` and mutate the output of the `.selector` selector.

Comment (blackfalcon) [reply documents/customProperties.md:40]: That's cool. Please make the suggestion and we can merge it in here.

**Review by jason-capsule42 — APPROVED** [review]

**Pull request merged**